portal/migrations/versions/4737a563e3f9_.py

```diff
@@ -34,6 +34,8 @@ def fixup_comorb(questionnaire_response_json, comorb_options):
     target_question = qnr_json_copy['group']['question'][0]
     coded_answers = []
     for answer in target_question['answer']:
+        if 'valueString' not in answer:
+            continue
```

Comment:
since the "fixed" answers will be pairs of valueCoding and valueString (maintained for legacy reasons), this will "double encode" answers that have already been fixed in migration.

it would be safest to return the original JSON if any valueCoding is found (none were used in the comorb instrument before it was fixed), implying the target QNR is already fixed.

Reply:
would you mind patching this up?